examples/advanced/bionemo/downstream/sabdab/run_sim_sabdab.py

```diff
@@ -22,7 +22,7 @@
 # job_name = "fedavg_sabdab_esm1nv"
 
 simulator = SimulatorRunner(
-    job_folder=f"jobs/{job_name}", workspace=f"/tmp/nvflare/results/{job_name}", n_clients=n_clients, threads=n_clients
+    job_folder=f"jobs/{job_name}", workspace=f"/tmp/nvflare/results/{job_name}", n_clients=n_clients, threads=1
```

Comment:
If we are changing the number of threads, then you should also note the hardware environment used. With larger GPU memory, you should be able to run more clients in parallel, i.e. `threads>1`

Reply:
Hi @holgerroth I would rather have one README.md instead of one for each folder. This one is found under the `examples/advanced/bionemo/README.md`.  I have addressed the changes with the double yaml in the paths and added in the initial note of the notebooks about changing the configs to handle multiple GPUs and the simulation scripts to handle multiple threads.

Reply:
Also the hardware used is in the first note appearing in each notebook as well as the bionemo version these notebooks work on.

Reply:
on the A1000, which is the gpu I have almost already gets quite heavy using threads > 1. It is fair to assume that people may have access to 1 gpu or with gpus not as performant. to make the demo more usable and accessible it is best to have demos with minimum valuable configurations so that more people do not run into problems. Those with better or more gpu can simply change the settings described.